x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/store/NativeRolesStoreTests.java

```diff
@@ -78,18 +82,19 @@ public void terminateThreadPool() throws Exception {
         terminate(threadPool);
     }
 
-    // test that we can read a role where field permissions are stored in 2.x format (fields:...)
-    public void testBWCFieldPermissions() throws IOException {
+    // test that we reject a role where field permissions are stored in 2.x format (fields:...)
+    public void testBWCFieldPermissions() throws Exception {
```

Comment:
I think we can remove this method now. It doesn't check backwards compatibility anymore (as we remove BWC in this PR) and we can get the assurance that we don't parse roles in 2x format in a unit test we can add in RoleDescriptorTests.java